<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WoW++ Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/editor.qrc">
    <normaloff>:/ico101.ico</normaloff>:/ico101.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="renderWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu_Project">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionLoadMap"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionUnit_Palette"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionExtractDBC"/>
    <addaction name="actionBuildNav"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionWoW_Editor_Info"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Modules</string>
    </property>
    <addaction name="actionObjectEditor"/>
    <addaction name="actionTriggerEditor"/>
   </widget>
   <addaction name="menu_Project"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuModules"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionObjectEditor"/>
   <addaction name="actionTriggerEditor"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionTranslate"/>
   <addaction name="actionRotate"/>
   <addaction name="actionScale"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadMap"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QDockWidget" name="unitPalette">
   <property name="windowTitle">
    <string>Unit Palette</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Units</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>World Objects</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="unitPaletteFilter">
       <property name="placeholderText">
        <string>Search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="unitPaletteView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionText">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionWoW_Editor_Info">
   <property name="text">
    <string>WoW++ Editor Info</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionObjectEditor">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Units.png</normaloff>:/Units.png</iconset>
   </property>
   <property name="text">
    <string>Object Editor</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Save.png</normaloff>:/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionLoadMap">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Open.png</normaloff>:/Open.png</iconset>
   </property>
   <property name="text">
    <string>Load Map</string>
   </property>
  </action>
  <action name="actionExtractDBC">
   <property name="text">
    <string>Extract DBC Files</string>
   </property>
  </action>
  <action name="actionBuildNav">
   <property name="text">
    <string>Build Navigation Mesh</string>
   </property>
  </action>
  <action name="actionTriggerEditor">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Trigger.png</normaloff>:/Trigger.png</iconset>
   </property>
   <property name="text">
    <string>Trigger Editor</string>
   </property>
   <property name="toolTip">
    <string>Trigger Editor</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Remove selected object(s)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionUnit_Palette">
   <property name="text">
    <string>Unit Palette</string>
   </property>
  </action>
  <actiongroup name="Movement">
   <action name="actionSelect">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="../qrc/editor.qrc">
      <normaloff>:/Select.png</normaloff>:/Select.png</iconset>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </action>
   <action name="actionTranslate">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="../qrc/editor.qrc">
      <normaloff>:/transformation.png</normaloff>:/transformation.png</iconset>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </action>
   <action name="actionRotate">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="../qrc/editor.qrc">
      <normaloff>:/rotationt.png</normaloff>:/rotationt.png</iconset>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </action>
   <action name="actionScale">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset resource="../qrc/editor.qrc">
      <normaloff>:/Scale.png</normaloff>:/Scale.png</iconset>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../qrc/editor.qrc"/>
  <include location="../qrc/style.qrc"/>
 </resources>
 <connections/>
</ui>
